examples/python/ghz.py

```diff
@@ -22,10 +22,10 @@
 import sys
 import os
 
-# We don't know from where the user is running the example,
-# so we need a relative position from this file path.
-# TODO: Relative imports for intra-package imports are highly discouraged.
-# http://stackoverflow.com/a/7506006
+ -# We don't know from where the user is running the example,
+ -# so we need a relative position from this file path.
+ -# TODO: Relative imports for intra-package imports are highly discouraged.
+ -# http://stackoverflow.com/a/7506006
```

Comment:
Thanks for reverting! Unfortunately, it seems that an extra ` -` slipped through at the beginning of each line, probably a result of some diffing. Could you please remove them?

If you are using `git` via the command line and your local repository contains a `master` branch that has not been modified locally, in cases like this it might be easier to just fetch the unchanged file from the master branch:
```
git checkout master -- examples/python/ghz.py
```